ideascube/monitoring/views.py

```diff
@@ -159,9 +159,9 @@ class InventoryDelete(DeleteView):
 class InventoryExport(CSVExportMixin, DetailView):
     model = Inventory
 
-    def get(self, request, *args, **kwargs):
+    def get(self, *args, **kwargs):
         self.object = self.get_object()
```

Comment:
Do we need to keep this assignation ?
We can use directly `self.get_object()` in `get_row` and `get_filename` and not redefined at all the `get` method.